fastats/scaling/scaling.py

```diff
@@ -0,0 +1,103 @@
+from numpy import empty_like, mean, std, sqrt, argsort, ones, nonzero, empty
```

Comment:
We leave one blank line at the start and end of `.py` files by convention throughout the project

Reply:
Good spot, will fix